meta-resin-common/recipes-core/systemd/systemd/resin.target

```diff
@@ -0,0 +1,2 @@
+[Unit]
+Requires=resin.target
```

Comment:
This should be in `multi-user.conf`, not here, but it doesn't seem correct. `multi-user.target` does not require `resin.target`. `resin.target` requires `multi-user`.

To do this properly the `default.target` should be symlinked to `resin.target`, and a `Requires=multi-user.target` should be added in this file.